app.js

```diff
@@ -0,0 +1,153 @@
+require('dotenv').config();
+
+let PORT = process.env.PORT;
+let express = require('express');
+let app = express();
+const MongoClient = require('mongodb').MongoClient;
+const MONGO_URI = process.env.MONGO_URI_MLAB;
+console.log(MONGO_URI);
+const bodyParser = require('body-parser');
+
+app.use(bodyParser.json());
+
+app.listen(PORT, () => {
+  console.log(`Server is up and running on port ${PORT}`);
+});
+
+app.get('/initialize', (req, res) => {
+  MongoClient.connect(MONGO_URI, (err, db) => {
+    if(err) {
+      // TODO handle db error
+      res.send('Error connecting to db' + err);
```

Comment:
should `return res.send`; remember that even though you terminated the response, the rest of the code would still run. This is true for all of the places where you do a res.send - whether it succeeded or failed.

Reply:
also, why initialize inside the request? If I hit the server 100 times will you make 100 connections to Mongo?

Reply:
Updated for return res.send( ... )
Also changed to single db connection